cpp/validate_harvested_records.cc

```diff
@@ -129,7 +138,8 @@ GeneralInfo GeneralInfo::Combine(const GeneralInfo &lhs, const GeneralInfo &rhs)
             combined_info.addField(*lhs_iter);
             ++lhs_iter;
         } else {
-            const int compare_result(StringUtil::AlphaWordCompare(lhs_iter->name_, rhs_iter->name_));
+            const int compare_result(StringUtil::AlphaWordCompare(lhs_iter->name_ + std::string(1, lhs_iter->subfield_code_),
```

Comment:
Not sure if i am happy with this hack even if it works in 99.99% of all cases

Reply:
Also, what about the RecordType?

Reply:
Is there still a problem here?

Reply:
What is wrong w/ the new code here?

Reply:
i will have a look at the re-implementation and close this one for now